.github/workflows/atex.yaml

```diff
@@ -0,0 +1,308 @@
+name: ATEX Testing Farm Integration
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+env:
+  ATEX_REPO: RHSecurityCompliance/atex-results-testing-farm
+  ATEX_HTML_REPO: RHSecurityCompliance/atex-html
+  CONTEST_REPO: RHSecurityCompliance/contest
+  ARTIFACT_RETENTION_DAYS: 1
+  TEST_TIMEOUT: 1440 # 24 hours
```

Comment:
24 hours seems unreasonable long to me, wouldn't 4 hours or so be better?

Reply:
This was mostly to avoid having to manually re-run tests when Testing Farm has an outage, or when there is high contention for resources during a (typical) 3-5 hour window. You can lower it down if you want, but then you might have to manually re-run more.